source/stylesheets/_circle.scss

```diff
@@ -62,11 +62,51 @@
   }
 }
 
+.circle-members-header {
+  text-align: center;
+  .circle-logo {
+    max-width: 60%;
+  }
+}
+
+.circle-members {
+  padding: 1em 0;
+  .circle-list-wrapper {
+    margin-bottom: 1em;
+    text-align: center;
+  }
+
+  .circle-title {
+    text-align: center;
+    text-transform: uppercase;
+    letter-spacing: 3px;
+    padding: .5em;
+    font-size: 1.5em;
+  }
+
+  .circle-list {
+    font-size: 1em;
+    line-height: 1.5em;
+  }
+
+  .yellow-star {
+    color: $yellow-tribune;
+    font-size: .8em;
+  }
+}
+
 
 .benefits-header {
   padding: 1em 0 0.5em 0;
 }
 
+.benefits-list {
```

Comment:
Agreed. Emily noted that with how it's currently presented, it wasn't easy to read. Before it didn't have the same spacing that it has on other pages and there was some capitalization confusion, so I wonder if I adjust it to match how it looks on the homepage if that would be easier rather than restyling all benefits lists across pages.